backend/inspirehep/records/cli.py

```diff
@@ -350,37 +349,61 @@ def legacy_records():
 
 
 @legacy_records.command(help="Upload marcxml column to S3.")
+@click.option(
+    "--task-queue",
+    help="Celery task queue",
+    default="xml_export",
```

Comment:
Harris wanted it to be flexible so we can have a special queue for it. I used `migrator` in the tests, so it will run without creating a new queue. I will change the default to `migrator` as well.

Reply:
It's ok, you can have a special queue, but then please test it with a new setting. Otherwise, it's not tested properly.